src/fund/components/FundButton.tsx

```diff
@@ -18,6 +19,7 @@ export function FundButton({
   target,
   text: buttonText = 'Fund',
 }: FundButtonReact) {
+  const componentTheme = useTheme();
```

Comment:
If there is time before the launch or as a fast follow - look to add Theme in an outer layer. Perhaps directly on `cn`